src/oracles/spot/lrt/WeEthWethSpotOracle.sol

```diff
@@ -41,7 +43,28 @@ contract WeEthWethSpotOracle is SpotOracle {
      * @return wethPerWeEth price of weETH in WETH. [WAD]
      */
     function getPrice() public view override returns (uint256) {
-        (, int256 ethPerWeEth,, uint256 ethPerWeEthUpdatedAt,) = BASE_WEETH_ETH_PRICE_CHAINLINK.latestRoundData(); // [WAD]
+        (
+            /*uint80 roundID*/
+            ,
+            int256 answer,
+            uint256 startedAt,
+            /*uint256 updatedAt*/
+            ,
+            /*uint80 answeredInRound*/
+        ) = BASE_SEQUENCER_UPTIME_FEED.latestRoundData();
+
+        if (answer == 1) revert SequencerDown();
```

Comment:
this looks good, did we want to implement the grace period check as well where we wait at least like 30 mins after back up?